packages/notifi-frontend-client/lib/storage/InMemoryStorageDriver.ts

```diff
@@ -32,6 +32,7 @@ export const createInMemoryStorageDriver = (
       keyPrefix += `:${config.walletPublicKey}`;
       break;
     }
+    case 'SUI':
```

Comment:
***CONCERN***
In this case, the accountAddress is the same as publicKey(authenticationKey). Not sure if it is correct to put in this group? or it would belong to the following group:
```
// ...
case 'ETHEREUM':
    case 'POLYGON':
    case 'ARBITRUM':
    case 'AVALANCHE':
    case 'BINANCE':
    case 'OPTIMISM':
    case 'SOLANA': {
      keyPrefix += `:${config.walletPublicKey}`;
      break;
    }
  // ...
```